client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx

```diff
@@ -142,6 +166,107 @@ class TransferDomainPrecheck extends React.PureComponent {
 		);
 	}
 
+	getRegistrationPeriodMessage() {
+		const { translate, domain } = this.props;
+		const {
+			creationDate,
+			expiryDateOk,
+			inInitialRegistrationPeriod,
+			loading,
+			termMaximumInYears,
+			transferEligibleDate,
+		} = this.state;
+
+		const dateFormat = translate( 'MMM D, YYYY', {
+			comment:
+				'Short month, day and year, like "Dec 5, 2017". This is a moment.js formatted string, ' +
+				'see http://momentjs.com/docs/#/displaying/format/',
+		} );
+
+		let heading = translate(
+			'{{strong}}%(domain)s{{/strong}} can be transferred in %(transferDelayInDays)s days.',
+			{
+				args: {
+					domain: domain,
+					transferDelayInDays: moment( transferEligibleDate ).diff( moment(), 'days' ),
+				},
+				components: {
+					strong: <strong />,
+				},
+			}
+		);
+
+		let message = translate(
+			"You don't have to wait though. Connect your domain to your site now, without transferring it. " +
+				'{{a}}Learn how{{/a}}.',
+			{
+				components: {
+					a: <a href={ support.MAP_EXISTING_DOMAIN } rel="noopener noreferrer" target="_blank" />,
+				},
+			}
+		);
+
+		let reason = null;
```

Comment:
I think the reason for why the transfer cannot be completed should stand out more. IMHO, the order of these two sentences should be reversed. Now it's:
<img width="738" alt="screen shot 2017-12-29 at 00 04 43" src="https://user-images.githubusercontent.com/3392497/34425413-e545e624-ec2b-11e7-8f3e-ea36799cbc0e.png">
"You can't transfer for x days, here's what you can do instead, here's why you can't transfer for x days, and here's the button to do something else instead". Seems a bit confusing to me :)

But if we swap them, we get a more logical flow, IMHO:
<img width="734" alt="screen shot 2017-12-29 at 00 05 57" src="https://user-images.githubusercontent.com/3392497/34425433-1cf46a82-ec2c-11e7-8282-62fcd237a27c.png">
"You can't transfer it for x days, here's exactly why, and here's a what you can do and a button to do it".

cc @danhauk